### PR TITLE
Implement admin api filer

### PR DESCRIPTION
**Pull request by hjmodi**

### CODE REVIEW

File: src/main/java/org/commonjava/indy/service/tracking/config/IndyTrackingConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import io.quarkus.runtime.Startup;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithName;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.File;
+
+@Startup
+@ConfigMapping( prefix = "folo" )
```
Comment (ligangty) [inline src/main/java/org/commonjava/indy/service/tracking/config/IndyTrackingConfiguration.java:26]: "folo" or "tracking"?

**Review by ligangty — COMMENTED** [review]
The code looks good, but I don't remember if we're still using this filer in folo tracking, because all folo data record has been migrated to use cassandra.
@ruhan1 @sswguo Do you remember if we are still using the folo data filer here?

**Comment by sswguo** [comment]
> The code looks good, but I don't remember if we're still using this filer in folo tracking, because all folo data record has been migrated to use cassandra. @ruhan1 @sswguo Do you remember if we are still using the folo data filer here?

We are using cassandra for the folo for a while, I think no need to consider it any more.

**Comment by hjmodi** [comment]
> > The code looks good, but I don't remember if we're still using this filer in folo tracking, because all folo data record has been migrated to use cassandra. @ruhan1 @sswguo Do you remember if we are still using the folo data filer here?
> 
> We are using cassandra for the folo for a while, I think no need to consider it any more.

it is used for creating a file here https://github.com/hjmodi/indy-tracking-service/blob/implement-listeners/src/main/java/org/commonjava/indy/service/tracking/controller/AdminController.java#L83 should I covert it to a single function instead?

**Comment by ligangty** [comment]
> > > The code looks good, but I don't remember if we're still using this filer in folo tracking, because all folo data record has been migrated to use cassandra. @ruhan1 @sswguo Do you remember if we are still using the folo data filer here?
> > 
> > 
> > We are using cassandra for the folo for a while, I think no need to consider it any more.
> 
> it is used for creating a file here https://github.com/hjmodi/indy-tracking-service/blob/implement-listeners/src/main/java/org/commonjava/indy/service/tracking/controller/AdminController.java#L83 should I covert it to a single function instead?

If the filer is just used for this single purpose, I think we can make it as simple as possible with a single function.

**Comment by hjmodi** [comment]
> > > > The code looks good, but I don't remember if we're still using this filer in folo tracking, because all folo data record has been migrated to use cassandra. @ruhan1 @sswguo Do you remember if we are still using the folo data filer here?
> > > 
> > > 
> > > We are using cassandra for the folo for a while, I think no need to consider it any more.
> > 
> > 
> > it is used for creating a file here https://github.com/hjmodi/indy-tracking-service/blob/implement-listeners/src/main/java/org/commonjava/indy/service/tracking/controller/AdminController.java#L83 should I covert it to a single function instead?
> 
> If the filer is just used for this single purpose, I think we can make it as simple as possible with a single function.

Cool, closing this PR. Will add this function when needed in another PR

**Pull request closed without merging**